### ISSUE TITLE
When dragging tabs from the sidebar it collapses, making it impossible to change a tree structure

### ISSUE THREAD

**Issue by Zaphkiell** [Shina-SG/Shina-Fox#19]
Floorp has the same issue with auto-collapse. With the left click pressed, the bar stops recognizing your hovering. Would be nice to find a fix, since it makes it really hard to use an otherwise amazing theme. And it's tailored for Floorp, no less!

**Comment by Shina-SG** [comment]
oh that's why you need `widget.gtk.ignore-bogus-leave-notify` set to `1`  and it would fix the whole thing according to this issue below https://bugzilla.mozilla.org/show_bug.cgi?id=1818517

**Comment by Shina-SG** [comment]
Hmm just to be sure, you're using the latest of Floorp on either Windows 10/11 or Linux and are using Sidebery right, if that is correct then try create a new profile to see if that work and if not then I might need more info to reproduce it and also attach a GIF to be sure we are in the same page as well

**Comment by Zaphkiell** [comment]
From about:support 
Name 	Floorp
Version 	11.8.1
OS 	Linux 6.7.0-arch3-1
Window Protocol	wayland
Desktop Environment hyprland
![ignore](https://github.com/Shina-SG/Shina-Fox/assets/72710853/893225bb-9dec-475c-aa90-493e780f7301)
![Recording](https://github.com/Shina-SG/Shina-Fox/assets/72710853/612d2a77-bf93-4ed9-b8f4-656d9fc3058b)
I managed to capture some of the weird behavior that seems random. Can't seem to reproduce reliably.

**Comment by Shina-SG** [comment]
the quality is fine, thanks and let's see can you try 2 more things:
1. Try it in a X11 session Floorp might not work well on Wayland sometime
2. Have the same setting on the stable branch of Firefox that's 122 and above on Wayland

**Comment by Zaphkiell** [comment]
I've launched floorp through xwayland and that seems to make the widget.gtk.ignore-bogus-leave-notify 1 work as intended. I haven't replicated the exact config on vanilla Firefox, however, I checked the code_sample from bugzilla and gtk.ignore doesn't work on it either. I would assume the problem with Wayland is similar there.

**Comment by Shina-SG** [comment]
right. thanks for that too, hmm so it's a wayland problem, I'm gonna pin this so people might see why it's not working and switch to x11 for their firefox until I can find a config that could fix the problem in wayland

**Comment by Zaphkiell** [comment]
Yeah. I'll go submit a bug to bugzilla. It seems to be separate from the one you posted and affects stable firefox. I don't want to open a separate issue, your guide has a minor error. 

browser.newtabpage.activity-stream.floorp.newtab.backdrop.blur.disable is a bool.

**Comment by Shina-SG** [comment]
oh yeah probably because I forgot to check, thanks

**Comment by LarveyOfficial** [comment]
Haven't personally had this issue on KDE 6 using Wayland. Only issue I have is the on hover stays activated when moving between monitors. (If window is on right monitor, moving mouse cursor to the left monitor causes the hover effect to stay activated, and the tabs to always be shows)
